elements/pfe-band/src/pfe-band.scss

```diff
@@ -306,7 +307,7 @@ $variables: (
   }
 
   &__aside {
-    align-self: flex-start;
+    align-self: stretch;
```

Comment:
@starryeyez024 I altered this because there's no way to make a card in the aside region fill the space unless the space is available to it in the first place.  The card needs to be set to `align-self: flex-start` to prevent it from filling the entire aside region.